appshell/cefclient.cpp

```diff
@@ -28,7 +28,7 @@ CefRefPtr<CefBrowser> AppGetBrowser() {
 
 CefWindowHandle AppGetMainHwnd() {
   if (!g_handler.get())
-    return NULL;
+    return 0;
```

Comment:
Why are we using an explicit integer for pointer here? CefWindowHandle is defined as HWND on Windows, NSView* on Mac OS X and GtkWidget* on Linux.

`NULL` seemed like a better choice. `nullptr` even better to handle type ambiguity with int.